requirements.txt

```diff
@@ -1,3 +1,4 @@
+attrs>=21.2.0
```

Comment:
I didn't set a good example, but would prefer concrete versions here to be used in CI

Reply:
https://github.com/facebookexperimental/usort/pull/68